geoips/plugins/modules/procflows/config_based.py

```diff
@@ -46,7 +47,7 @@
         write_stats_to_database,
     )
 except ImportError:
-    print("Please install geoips_db package if required")
+    pass
```

Comment:
Should these go to debug or info or something? It seems problematic if they silently pass.

Reply:
Pretty sure flake8 will complain if you run the command to set up logging before imports.  What is generally done for catching failed imports? 

LOG = logging.getLogger(__name__)

Reply:
If it becomes an issue we could also tell flake8 to just ignore these lines? For now, fine.